db/migrate/20160223195539_migrate_network_partner_participants_to_facilitators.rb

```diff
@@ -0,0 +1,34 @@
+class MigrateNetworkPartnerParticipantsToFacilitators < ActiveRecord::Migration
```

Comment:
I'm not sure what you're asking, we agreed on a rollback strategy for staging and production and you're seeing it here.

Worst case scenario, the network partners will still have access to assessments but as facilitators instead of participants.